tests/test_devices.py

```diff
@@ -0,0 +1,213 @@
+# Copyright (c) 2023 - 2025 Chair for Design Automation, TUM
+# Copyright (c) 2025 Munich Quantum Software Company GmbH
+# All rights reserved.
+#
+# SPDX-License-Identifier: MIT
+#
+# Licensed under the MIT License
+
+"""Test the IBM devices."""
```

Comment:
The docstring is not correct here. Also, I suggest to rename the file to `test_targets` since also the native gatesets are validated here.

Some further thoughts:
- Should we cover all available devices here? Right now, this is the case for IBM but not for IonQ.
- Can we refactor the tests to avoid code duplication? One problem might be that some devices provide additional duration data.
- The test function names should be streamlined as well. Right now, some are still mentioning `from_calibration`.